fee_allocator/fees_collected/fees_2024-05-23_2024-06-06.json

```
@@ -1,6 +1,6 @@
 {
   "mainnet": 377313.85,
```

```suggestion
  "mainnet": 322357.85
```

Mainnet also changed when Arbi updated as it just seems to be based off subtacting out other chains?